src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```
you should instead create a global ColorSupplier field here and use it inside of the method